src/main/java/com/cdc/apihub/mx/employmentVerification/client/model/EmploymentVerifications.java

```diff
@@ -1,50 +1,44 @@
 
-
 package com.cdc.apihub.mx.employmentVerification.client.model;
 
-import java.util.Objects;
-
-import com.cdc.apihub.mx.employmentVerification.client.model.EmploymentVerification;
-
 import java.util.ArrayList;
-
-
+import java.util.Objects;
 
 @javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.JavaClientCodegen", date = "2021-01-12T10:06:28.689207-06:00[America/Mexico_City]")
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.